emails/components/TailwindConfig.ts

```diff
@@ -0,0 +1,40 @@
+const defaultTheme = require('tailwindcss/defaultTheme');
+
+export const TailwindConfig = {
+  theme: {
+    extend: {
+      height: {
+        18: '4.35rem',
+        17: '4.3rem',
+        'screen-60': '80vh',
+      },
+      colors: {
+        bcblue: '#003366',
+        bcorange: '#FCBA19',
+        darkergrey: '#344054',
+        mediumgrey: '#475467',
+        tableheadergrey: 'rgba(214, 216, 213, 0.15)',
+        tablebordergrey: '#EAECF0',
+        disabledborder: 'D0D5DD',
+        cloudgrey: '#667085',
+        divider: '#0000001f',
```

Comment:
Whatever the case may be, let's keep consistent case conventions. I've noticed that all the color hash strings elsewhere are in uppercase.